cabal-install/src/Distribution/Client/Init/Types.hs

```diff
@@ -186,6 +191,20 @@ data TestTarget = TestTarget
     , _testBuildTools :: [P.Dependency]
     } deriving (Show, Eq)
 
+-- | 'CommonTarget' represents the common options that the user
+-- can import within the other targets.
+--
+data CommonTarget = CommonTarget
+    { _commonName   :: String
+    , _commonFields :: [CommonField]
+    } deriving (Show, Eq)
+
+data CommonField = CommonField
+    { _commonFieldName    :: String
+    , _commonFieldValue   :: String
+    , _commonFieldComment :: [String]
+    } deriving (Show, Eq)
+
```

Comment:
@emilypi what are your thoughts? While this makes using multiple, separate, common stanzas trivial within `cabal init`, I'm not sure we really have the need for it.

Reply:
I don't think we need it even if it's trivial, but if you want to add in `-Wall` support and showcase them, be my guess.

Reply:
Yeah... I've been pondering, it adds complexity to the codebase; even though it's somewhat minimal right now, every little change/tuning/addition sums up and soon enough we'll be with a happy mess in our hands. The future proofing and flexibility that this change would provide (regarding common stanzas) is niftily nice, but I don't think it's worth the extra code that it incurs throughout the cabal init codebase for what it brings to the table. I'll likely do as you suggest and streamline the warnings' common stanzas to a hardcoded approach.

Reply:
Sounds good - :+1: 

Yeah, what i was thinking about was that in order to make this worthwhile, you'd probably also want to throw the base dep in there, but then the process of aggregating dependencies becomes a bit more tiresome, and you end up having to fetch the base dep early on, and then pass it through to all of the targets as a parameter, or you have to "fixup" the dependencies in the `createProject` function like we do when we add the library dep to the exe and test suite targets. It starts to become more tangled, and not for any tangible benefit aside from the showcase.